src/components/controls/Stream.tsx

```diff
@@ -13,13 +13,30 @@ export interface IStreamProps {
 export interface IStreamState { };
 
 export class Stream extends React.PureComponent<IStreamProps, IStreamState> {
+    videoRef: HTMLVideoElement;
+
+    public componentDidMount() {
+        this.videoRef.srcObject = this.props.stream;
+    }
+
+    public componentDidUpdate(nextProps: IStreamProps) {
+        if (nextProps.stream !== this.props.stream) {
+            // NOTE(denggl): Not running here now.
+            // But in future, maybe it will switch the stream and reload the video.
+            this.videoRef.srcObject = this.props.stream;
+        }
+    }
+
     public render() {
         const { stream, className } = this.props;
-        const srcURL = URL.createObjectURL(stream);
 
         return (
             <div className={classnames(this.props.className, 'rcw-stream')}>
-                <video className='rcw-stream-video' autoPlay={true} src={srcURL} />
+                <video
+                    className='rcw-stream-video'
+                    ref={(node: HTMLVideoElement) => this.videoRef = node}
```

Comment:
The `refVideo` method and `videoRef` element seem weird.
I want to use `videoElement` instead of `videoRef`. That's ok?